akka-remote/src/test/scala/akka/remote/RemotingSpec.scala

```diff
@@ -361,8 +379,9 @@ class RemotingSpec extends AkkaSpec(RemotingSpec.cfg) with ImplicitSender with D
       // msg to old ActorRef (different uid) should not get through
       child2.path.uid should not be (child.path.uid)
       child ! 46
-      expectNoMsg(1.second)
-      system.actorFor(system / "looker1" / "child") ! 47
+      expectNoMessage(1.second)
+      @silent
+      val ___ = system.actorFor(system / "looker1" / "child") ! 47
```

Comment:
why the __ val?

Reply:
Because `_` is already taken above :)

Reply:
(and without a variable declaration it's not a valid place to place `@silent`)

Reply:
I think this is somewhat ridiculous for tests, can't we just disable the warnings for tests instead?

Reply:
I've picked up various tests that don't test what they mean to with warning. Can we just be very liberal with a silent on an entire test class rather than disable them completely?

Reply:
Agreed, I like having fatal warnings for tests by default, but I'm fine with using `@silent` loosely there.